physics/dynamic_frame_test.cpp

```diff
@@ -0,0 +1,71 @@
+#include "physics/dynamic_frame.hpp"
+
+#include "geometry/frame.hpp"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "quantities/named_quantities.hpp"
+#include "testing_utilities/almost_equals.hpp"
+#include "testing_utilities/componentwise.hpp"
+
+namespace principia {
+
+using geometry::Frame;
+using geometry::InnerProduct;
+using quantities::GravitationalParameter;
+using quantities::Sqrt;
+using testing_utilities::AlmostEquals;
+using testing_utilities::Componentwise;
+
+namespace physics {
+
+namespace {
+
+using Circular =
+    Frame<serialization::Frame::TestTag, serialization::Frame::TEST, true>;
+using Helical =
+    Frame<serialization::Frame::TestTag, serialization::Frame::TEST1, true>;
+
+Vector<Acceleration, Circular> Gravity(Instant const& t,
+                                       Position<Circular> const& q) {
+  Displacement<Circular> const r = q - Circular::origin;
+  auto const r2 = InnerProduct(r, r);
```

Comment:
That would just hide an additional division though, and I'm not sure it improves clarity.